config/manager/manager.yaml

```
@@ -35,7 +35,7 @@ spec:
         - --leader-elect
         - -zap-log-level=info
         - -enable-webhooks=false
-        image: redis-operator:e2e
+        image: controller
         imagePullPolicy: IfNotPresent
```

```suggestion
        imagePullPolicy: Never
```

Can you make this Never I want to check which version it is using